WordPress/gutenberg
# Update minimum required version in PHP.

## What?

Updates the minimum required version in the admin notices and deactivation hooks.

## Why?

To match the minimum version in the plugin header for consistency.

## How?

* Introduces a constant for the minimum required WordPress version
* Updates the minimum version checks to use said constant
* Introduces a unit test to ensure the plugin header and new constant match

Let me know if you'd like me to define the value elsewhere or avoid using a constant.

## Testing Instructions

1. Install a version of WordPress 6.4
2. Visit the dashboard
3. Ensure that the plugin deactivates and a notice is shown to the user.

### Testing Instructions for Keyboard


## Screenshots or screencast
